mcmd/commands/enable.py

```diff
@@ -23,6 +24,16 @@ def arguments(subparsers):
                               type=str,
                               help="The entity type to secure")
 
+    p_enable_theme = p_enable_subparsers.add_parser('theme',
+                                                    help='Enables the bootstrap theme which changes the styling of your'
+                                                         ' MOLGENIS')
+    p_enable_theme.set_defaults(func=enable_theme,
+                                write_to_history=True)
+    p_enable_theme.add_argument('theme',
+                                metavar='THEME',
+                                type=str,
+                                help='The bootstrap theme you want to enable (without .css)')
```

Comment:
I think this should be expanded a bit now there are multiple 'correct' ways to choose a theme.